v20.2/create-user.md

```diff
@@ -61,6 +68,248 @@ Secure clusters require users to authenticate their access to databases and tabl
 
 ## Examples
 
+To run the following examples, [start a secure single-node cluster](cockroach-start-single-node.html) and use the built-in SQL shell:
```

Comment:
Not suggesting this for this PR, but gosh it's a shame there are so many "almost the same" variants of these commands.  It would be nice to "templatize" this using Jekyll/Liquid as mentioned in an earlier comment.  Unfortunately that would probably be too fragile anyway due to subtle variations.